### ISSUE TITLE
Set locale and persist it in the path

### ISSUE THREAD

**Issue by Phazz** [activeadmin/activeadmin#2178]
I am trying to find a way to change locale in an Active Admin (0.6.0) app I made. I have two links, one for each language that sets properly the locale parameter in the path. I do it this way in my active_admin.rb :

``` ruby
config.namespace :app do |app|
  app.build_menu :utility_navigation do |menu|
  menu.add :label => "English",:url => proc { url_for(:locale => 'en') }, id: 'i18n-en'
  menu.add :label => "Spanish",:url => proc {  url_for(:locale => 'es') }, id: 'i18n-es'
  menu.add :label => proc{ display_name current_active_admin_user},
            :url => '#',
            :id => 'current_user',
            :if => proc{ current_active_admin_user? }
  app.add_logout_button_to_menu menu
end
```

Now for the routing part I have in routes.rb :

``` ruby
root :to => 'app/dashboard#index'    
scope ":locale", locale: "#{I18n.locale}" do
  resources :customers
  ActiveAdmin.routes(self)
  devise_for :admin_users, ActiveAdmin::Devise.config
end    
match '*path', to: redirect("/#{I18n.locale}/%{path}"), constraint: lambda { |req| !req.path.starts_with? "/#{I18n.locale}/"}
match '', to: redirect("/#{I18n.locale}/")
```

With this configuration I get the expected result, when I click English my path change to /en/path and on Spanish /es/path. Fine.

Active Admin create himself its routes, ActiveAdmin.routes(self). So links for resources in the bar, for example Customer, link to /es/customer (es is my default locale). So even when I set the locale to en using my menu, when I click a resource the locale change to es.

In short, I want to persist the locale selected with the menu.

Anyone could point me in the right direction to do this right ?

Thanks in advance.
